tests/component/encapsulation.cpp

```diff
@@ -282,7 +283,14 @@ TEST(Encapsulation, parseAlternateFormHierarchy)
     libcellml::ModelPtr model = parser.parseModel(input);
 
     EXPECT_EQ(size_t(0), parser.errorCount());
+    printErrors(parser);
```

Comment:
What's the point of printing the errors if we don't check what is being printed?
```suggestion
    printErrors(parser);
```
```suggestion
    printErrors(parser);
```

Reply:
That was a debug statement that I didn't remove, removing it now.